### PR TITLE
chore(core): remove v1 settings 

### PR DESCRIPTION
**Pull request by kyleleow**
#### What kind of PR is this?:

/kind chore

#### What this PR does / why we need it:
- Remove v1 screens used in settings navigator
- Rename v2 screens used in settings navigator to v1

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
- After deleting v1 files, I used `git mv` command to change the file name of v2 to v1, . So on the file changed, it may looks like i edited v1 but actually it is not.

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [x] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx [hunk 1]
```diff
@@ -42,33 +44,39 @@ export function ConfirmPinScreen ({ route }: Props): JSX.Element {
...
     const copy = { words, network, pin }
-    setSpinnerMessage(translate('screens/PinConfirmation', 'It may take a few seconds to update your passcode...'))
+    setSpinnerMessage(translate('screens/PinConfirmation', 'It may take a few seconds to update your passcode.'))
     setTimeout(() => {
       MnemonicEncrypted.toData(copy.words, copy.network, copy.pin)
         .then(async encrypted => {
           await MnemonicStorage.set(words, pin)
-          await setWallet(encrypted)
-          navigation.dispatch(StackActions.popToTop())
+          await passcodeChangeSuccess(encrypted)
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx:52]: How would this await work if the function is not async?

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx [hunk 2]
```diff
@@ -42,33 +44,39 @@ export function ConfirmPinScreen ({ route }: Props): JSX.Element {
...
         })
         .catch(logger.error)
     }, 50) // allow UI render the spinner before async task
   }
 
+   function passcodeChangeSuccess (encrypted: any): void {
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx:58]: type for Encrypted

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx (hunk 1 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx]: removed `await`

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx (hunk 2 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx]: added type

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**